sml_exporter/__init__.py

```diff
@@ -168,6 +172,27 @@ def event(self, message_body: SmlSequence) -> None:
             logger.debug(
                 "Vendor or device identifiers not initialized, event was ignored."
             )
+        self.reset_watchdog()
+
+    def clear_gauges(self):
+        logger.warning("Timeout on receiving serial data, clearing gauges.")
```

Comment:
We should probably talk about metrics here and not specify the concrete type.

Reply:
Indeed, fixed.